easybuild/easyblocks/d/dirac.py

```diff
@@ -54,7 +54,7 @@ def configure_step(self):
                 raise EasyBuildError("Failed to remove existing install directory %s: %s", self.installdir, err)
 
         self.cfg['separate_build_dir'] = True
-        self.cfg.update('configopts', "-DENABLE_MPI=ON -DCMAKE_BUILD_TYPE=release")
+        self.cfg['build_type'] = 'release'  # TODO: Verify if this is really lowercase
```

Comment:
As far as I know the values passed to CMake options are case-insensitive (but you should confirm that)

Reply:
Not always. If it is used directly casing does matter (in a strequal e.g.)
I verified the other uses but can't for this one as access to source requires a license